handbook/company/pricing-features-table.yml

```diff
@@ -473,6 +477,7 @@
   documentationUrl:
   tier: Premium
   comingSoonOn: 2024-06-30
+  isExperimental: yes
```

Comment:
```suggestion
```

Not experimental yet b/c it's not shipped.

Also, I'm not sure why there's a `comingSoonOn` on this one. That's usually reserved for customer commits/promises.

@zayhanlon do you know if we committed "Asset discovery" to anyone?

Reply:
Assuming no, I opened a PR here to bump it to 2025: https://github.com/fleetdm/fleet/pull/20809

Reply:
@noahtalerman no idea - sounds like the 'discovery probe to find all devices in an environment' think we may have been discussing with Deloitte? i wasn't here when this was added on and not sure what it was referring to but no one is waiting on this as far as i'm aware